Backend/DTOs/UserDTO.cs

```diff
@@ -0,0 +1,13 @@
+namespace Backend.DTOs;
+
+public class UserDTO
+{
+    public int Id { get; set; }
+    public string Email { get; set; } = null!;
+    public string Username { get; set; } = null!;
+    public string FirstName { get; set; } = null!;
+    public string LastName { get; set; } = null!;
+    public int? BirthYear { get; set; }
+    public string Password { get; set; } = null!;
+    public string APIKey { get; set; } = null!;
```

Comment:
I was thinking maybe we can call this something else... Like authToken or whatever... as it is not practicly key